nginx-agent.conf

```diff
@@ -38,4 +38,8 @@ config_dirs: "/etc/nginx:/usr/local/etc/nginx:/usr/share/nginx/modules:/etc/nms"
 
 api:
   # default port for NGINX Agent API, this is for the server configuration of the REST API
-  port: 8081
+  # port: 8081
```

Comment:
Is this the default port? Should we change this to a different suggestion, or leave it blank, to force users to define a different value?